tiledb/sm/cpp_api/query.h

```diff
@@ -529,7 +523,9 @@ class Query {
    * @return Reference to this Query
    */
   template <class T>
-  Query& add_range(uint32_t dim_idx, T start, T end, T stride = 0) {
+  TILEDB_DEPRECATED  // Or maybe shouldnot be???
```

Comment:
I think this is correct, we should deprecate the old-style of setting ranges. Remove the in-code comment.

Reply:
questioning comment removed